### DISCUSSION TITLE
When sending logs from td-agent Fluentforwardreceiver throwing error EOF connection closed

### DISCUSSION THREAD

**Discussion by pladynski** [open-telemetry/opentelemetry-collector-contrib#24775] [Q&A]
Hi All and @dmitryax ,

I am trying to receive td-agent logs through fluentfarwardreceiver and cannot get it to work. I try to send logs from latest td-agent to otel-contrib 0.82.0 and I tried from Linux, from Windows and to two different target linux rhel machines and each time I get such output on otel console:

![image](https://github.com/open-telemetry/opentelemetry-collector-contrib/assets/6035560/b4d7ef05-3d7d-4e31-bc05-2790b883300b)

Its like receiving connection and instantly closing it. My config in otel is super easy
![image](https://github.com/open-telemetry/opentelemetry-collector-contrib/assets/6035560/66610ee7-6cfd-470f-ad8d-d1a734836ee5)

And my td-agent config is even simpler:
![image](https://github.com/open-telemetry/opentelemetry-collector-contrib/assets/6035560/4a5e8526-7572-45c1-93ba-9ff8a4c9e074)

What can be the root cause of such behavior? Can you share with me any working config samples you are using?

**Comment by karthikranganathan85** [comment]
Hi,

I am facing the similar issue while forwarding the logs from Fluentd to Splunk Otel collector through fluentforward receiver. Can anyone please help us?

**Comment by dimitrisfinas** [comment]
same for me. No clear explanations on why it happens. Did anybody solved this?
After searching, I found this: https://github.com/fluent/fluentd/issues/2642 and maybe our issue is linked to same root cause, using secured TLS sending of fluent events, while collector is only supporting not secured for the moment

**Reply by payxn** [reply]
Hello,

I thought the same. But even without using TLS with the forward plugin, it's not working...
